main.js

```diff
@@ -0,0 +1,17 @@
+function safeCall(f) {
+    try {
+        f();
+        return true;
+    } catch (e) {
+    }
+    return false;
+}
+
+Function.prototype.safeCall = function() {
+    try {
+        this();
+        return true;

```

Comment:
safeCall in prototype should have the same behavior as call

Reply:
"the same behavior as call" means function `safeCall` should accept context, arguments and call the original function with them